web_widget_json_graph/static/src/js/web_widget_json_graph.js

```diff
@@ -0,0 +1,55 @@
+odoo.define('web.web_widget_json_graph', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var AbstractField = require('web.AbstractField');
+    var field_registry = require('web.field_registry');
+    var QWeb = core.qweb;
```

Comment:
@luisg123v You can also remove unused Qweb variable.

Reply:
Done, also removed core as it's neither used.